### PR TITLE
Updated the ReadMe File and fixed the Navigatior bar width

### PR DESCRIPTION
**Pull request by Navdeep-av**
Updated the ReadMe File and fixed the Navigatior bar width

### CODE REVIEW

File: client/src/components/organisms/Navbar/NavBar.Component.jsx [hunk 1]
```diff
@@ -9,18 +9,27 @@ export default function NavBar() {
...
+          />
+
+          {/* <Navbar.Brand>Code Lele</Navbar.Brand> */}
 
           <Nav className="me-auto">
             <Link
               className="p-2"
-              style={{ textDecoration: "none", color: "white" }}
+              style={{
+                textDecoration: "none",
```
Comment (xDAnkit) [inline client/src/components/organisms/Navbar/NavBar.Component.jsx]: @Navdeep-av - Use styled components

File: client/src/testing.jsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export default function EnvTest() {
+  console.log("Domain:", import.meta.env.VITE_AUTH_DOMAIN);
```
Comment (xDAnkit) [inline client/src/testing.jsx]: @Navdeep-av - Delete this file

**Review by xDAnkit — CHANGES_REQUESTED** [review]
Pls review comments

**Comment by Navdeep-av** [comment]
I have made the changes, please review.

File: client/public/css/styles.css [hunk 3]
```diff
@@ -45,15 +45,21 @@ p {
 
 .ProfileName {
   color: white;
+  margin-bottom: 0;
+  font-weight: 600;
 }
 
+.logout-btn{font-weight: 600 !important;}
```
Comment (xDAnkit) [inline client/public/css/styles.css]: @Navdeep-av - Remove if not in use

File: client/public/css/styles.css [hunk 4]
```diff
@@ -45,15 +45,21 @@ p {
...
+
+.nav-width{max-width: 100% !important;}
```
Comment (xDAnkit) [inline client/public/css/styles.css]: @Navdeep-av - Format this with prettier

File: client/public/css/styles.css [hunk 5]
```diff
@@ -45,15 +45,21 @@ p {
...
+
 .btn-link {
   text-decoration: none;
   color: black;
 }
 
 .code-logo {
-  width: 30px;
+  /* width: 30px; */
```
Comment (xDAnkit) [inline client/public/css/styles.css]: @Navdeep-av - Remove commented code

File: client/src/main copy.jsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
+import { createRoot } from "react-dom/client";
```
Comment (xDAnkit) [inline client/src/main copy.jsx]: @Navdeep-av - Remove unnecessary fille

**Review by xDAnkit — CHANGES_REQUESTED** [review]
Pls check it

File: client/public/css/styles.css (hunk 3 above)
Comment (Navdeep-av) [reply client/public/css/styles.css]: @xDAnkit Fixed.

File: client/public/css/styles.css (hunk 5 above)
Comment (Navdeep-av) [reply client/public/css/styles.css]: @xDAnkit Done.

File: client/public/css/styles.css (hunk 4 above)
Comment (Navdeep-av) [reply client/public/css/styles.css]: @xDAnkit Fixed.

File: client/src/main copy.jsx (hunk 6 above)
Comment (Navdeep-av) [reply client/src/main copy.jsx]: @xDAnkit Done

**Review by xDAnkit — APPROVED** [review]
Looks good to me

**Pull request merged**